flake.nix

```diff
@@ -41,20 +41,10 @@
           "rustfmt"
           "llvm-tools-preview"
         ];
-        fenixStable = fenix.packages.${system}.combine [
-          ( fenix.packages.${system}.stable.withComponents [
-              "cargo"
-              "clippy"
-              "rust-src"
-              "rustc"
-              "llvm-tools-preview"
-            ]
-          )
-          ( fenix.packages.${system}.latest.withComponents [
-              "rustfmt"
-            ]
-          )
```

Comment:
I think this means we won't have nightly `rustfmt` in the Nix flake, so the settings in `rustfmt.toml` won't work. Could we do e.g.

```
        fenixStable = fenix.packages.${system}.combine [
        (
        fenix.packages.${system}.latest.withComponents [
          "rustfmt"
        ]
        )
        ( fenix.packages.${system}.fromToolchainFile {
          dir = ./.;
          sha256 = "sha256-opUgs6ckUQCyDxcB9Wy51pqhd0MPGHUVbwRKKPGiwZU=";
        } )

        ];

``` 

(excuse the formatting)

We could also set this in the `rust-toolchain.toml` file if possible, though I don't know exactly how that works

Reply:
Duh, I forgot why we used that. I don't think you can mix-and-match toolchain components with rustup, but we certainly can retain nightly rustfmt in the flake.